components/infobox/wikis/ageofempires/infobox_league_custom.lua

```diff
@@ -35,16 +35,17 @@ local CustomInjector = Class.new(Injector)
 local _league
 local categories = {}
 
-local _TIER_SHOW_MATCH = 9
-
 function CustomLeague.run(frame)
 	local league = League(frame)
 	_league = league
+
+	league.args.liquipediatier = Tier.toNumber(league.args.liquipediatier)
```

Comment:
```suggestion
	league.args.liquipediatier = tostring(Tier.toNumber(league.args.liquipediatier))
```
Needed until Module:MetadataGenerator is moved to use new tier stuff.

Reply:
needs to be done that way in the other customs that use this function then too

Reply:
updating metadata gen instead